app/src/main/res/values-it/strings.xml

```diff
@@ -122,100 +122,162 @@
-    <string name="look_and_feel_theme_system_black">Di sistema nero</string>
+    <string name="look_and_feel_theme_system">Di Sistema</string>
+    <string name="look_and_feel_theme_system_black">Di Sistema Nero</string>
     <string name="look_and_feel_theme_light">Chiaro</string>
     <string name="look_and_feel_theme_dark">Scuro</string>
     <string name="look_and_feel_theme_black">Nero</string>
-    <string name="look_and_feel_theme_color">Colore tema</string>
-    <string name="look_and_feel_post_view">Visualizzazione post</string>
+    <string name="look_and_feel_theme_color">Colore del Tema</string>
+    <string name="look_and_feel_post_view">Formato Post</string>
     <string name="look_and_feel_post_view_card">Tessera</string>
-    <string name="look_and_feel_post_view_small_card">Tessera piccola</string>
+    <string name="look_and_feel_post_view_small_card">Tessera Piccola</string>
     <string name="look_and_feel_post_view_list">Lista</string>
-    <string name="look_and_feel_show_navigation_bar">Mostra barra di navigazione</string>
+    <string name="look_and_feel_show_navigation_bar">Mostra barra di navigazione </string>
     <string name="look_and_feel_use_custom_tabs">Usa schede personalizzate</string>
     <string name="settings_activity_settings">Impostazioni</string>
-    <string name="settings_activity_look_and_feel">Aspetto</string>
+    <string name="settings_activity_look_and_feel">Aspetto Interfaccia</string>
     <string name="settings_activity_account_settings">%1$s impostazioni</string>
-    <string name="settings_activity_about">A proposito di</string>
-    <string name="community_blocked">%1$s Bloccate</string>
-    <string name="comment_mention_node_permalink_copied">Permalink copiato</string>
-    <string name="person_routines_blocked">%1$s bloccati</string>
-    <string name="utils_login_first">Effettua prima il login</string>
-    <string name="login_view_model_incorrect_login">Login errato</string>
+    <string name="settings_activity_about">Info</string>
+    <string name="community_blocked">%1$s Bloccati</string>
+    <string name="comment_mention_node_permalink_copied">Permalink Copiato</string>
+    <string name="person_routines_blocked">%1$s Bloccati</string>
+    <string name="utils_login_first">Effettua prima il Login</string>
+    <string name="login_view_model_incorrect_login">Login Errato</string>
     <string name="login_view_model_is_not_a_lemmy_instance">%1$s non è un\'istanza Lemmy</string>
-    <string name="create_report_view_model_report_created">Report creato</string>
+    <string name="create_report_view_model_report_created">Segnalazione Creata</string>
     <string name="about_social">Social</string>
     <string name="app_bars_dot_spacer">·</string>
     <string name="comment_node_in">"in "</string>
     <string name="look_and_feel_activity_show_content_for_collapsed_comments">Mostra contenuto per commenti compressi</string>
```

Comment:
I know I provided the initial translation, but now I'm wondering whether this alternative is preferable:
Mostra Contenuto dei Commenti Minimizzati

Reply:
I agree, I think we can even remove "contenuto". "Mostra commenti minimizzati" I think gets the point across while being shorter.